smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -48,7 +48,7 @@
 import dragon.native.machine as dragon_machine
 
 from smartsim._core.launcher.dragon.pqueue import NodePrioritizer, PrioritizerFilter
-from smartsim._core.mli.infrastructure.control.event_listener import (
+from smartsim._core.mli.infrastructure.control.listener import (
```

Comment:
Im having some trouble commenting on this file below:

1. DragonStatus could have a docstring -> Statuses for a Dragon process
2. ProcessGroupInfo.hosts attr docstring is incomplete
3. DragonBackend.start_event_listener mentions CPU affinity for both CPU and GPU

Reply:
- DragonStatus is broken and will be removed completely in a ticket very soon
- fixed
- fixed